icons/wallet.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 6v12c0 1.1.9 2 2 2h16v-4M3 6c0 1.1.9 2 2 2h14M3 6c0-1.1.9-2 2-2h14v4m0 0h2v4M16 14c0-1.1.9-2 2-2h4v4h-4a2 2 0 0 1-2-2z" />
+</svg>
```

Comment:
Updated the paths to draw correctly in Figma

![image](https://user-images.githubusercontent.com/690573/79478271-33170980-7fd9-11ea-9fc9-a54535b4fab0.png)

```suggestion
<svg width="21" height="18" viewBox="0 0 21 18" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M1 3V15C1 16.1 1.9 17 3 17H19V13M1 3C1 4.1 1.9 5 3 5H17M1 3C1 1.9 1.9 1 3 1H17V5M17 5H19V9M14 11C14 9.9 14.9 9 16 9H20V13H16C15.4696 13 14.9609 12.7893 14.5858 12.4142C14.2107 12.0391 14 11.5304 14 11Z" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

Reply:
The viewBox is 21x18px here. Needs to be a 24px frame